.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
Seems this can be picked up by Visaul Studio, do we want LF on Windows? IIRC, nost git clones will do CRLF...

Reply:
The editor might not be the immediate problem here. If you change file EOL, git will see the whole file as changed.

Reply:
AFAIK everything is already LF.

Reply:
yeah.

find -name "*.cpp" -exec dos2unix '{}' \;

doesn't result in any changes.

Reply:
Depends on how someone's git is set up on Windows (see core.autocrlf), could force CRLF on clone...

Reply:
If I do your line above, this is my `git status`:
```
$ git status
Refresh index: 100% (1934/1934), done.
On branch main
Your branch is up to date with 'upstream/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   app/actions.cpp
        modified:   app/app_utils.cpp
        modified:   app/exiv2.cpp
        modified:   app/getopt.cpp
        modified:   app/wmain.cpp
        modified:   contrib/Qt/main.cpp
        modified:   fuzz/fuzz-read-print-write.cpp
        modified:   samples/addmoddel.cpp
...
```

Reply:
hrm I'll look into this.

Reply:
@kmilos removed.